pkg/test/echo/server/forwarder/protocol.go

```diff
@@ -197,7 +197,7 @@ func newProtocol(cfg Config) (protocol, error) {
 			}
 		} else if cfg.Request.Http2 && scheme.Instance(urlScheme) == scheme.HTTPS {
 			if cfg.Request.Alpn == nil {
-				tlsConfig.NextProtos = []string{"h2"}
+				tlsConfig.NextProtos = []string{"h2", "http/1.1"}
```

Comment:
> As an exception, when the value "h2" is included in the server's Config.NextProtos, HTTP/1.1 clients will be allowed to connect as if they didn't support ALPN. See issue 46310 for more information.

I thought this is not needed here? And the problem is that `istio-peer-exchange` is rejected